### ISSUE TITLE
failing litehouse accessibility

### ISSUE THREAD

**Issue by JeromeDaGnome** [JeromeDaGnome/brixicon#1]
![image](https://user-images.githubusercontent.com/11600460/162549184-bdb4443d-7bba-46c2-af8b-a803ecbdfe0b.png)
![image](https://user-images.githubusercontent.com/11600460/162549271-b2b5427f-051d-4b68-bdba-4febcb0e194b.png)

1. contrast issues
2. alt text on images
3. links don't have discernable labels

**Comment by JeromeDaGnome** [comment]
![image](https://user-images.githubusercontent.com/11600460/162666446-596a5077-f24e-4a9c-a1c9-bf72892720b9.png)

fixed content and alt images.  added aria labels to all collapsed links.
